shared/chat/conversation/rekey/participant-rekey.native.tsx

```diff
@@ -17,7 +17,11 @@ const Row = ({username, onUsernameClicked}) => (
       <Avatar username={username} size={48} style={{marginRight: globalMargins.small, padding: 4}} />
       <Box style={innerRowStyle}>
         <Usernames inline={true} backgroundMode="Terminal" type="BodySemibold" users={[{username}]} />
-        <Text type="BodySmall" negative={true} style={{color: globalColors.blueLighter_40, lineHeight: 17}}>
+        <Text
+          type="BodySmall"
+          negative={true}
+          style={{color: globalColors.blueLighter_40, lineHeight: 17} as any}
```

Comment:
Why does it need any here? If the color isn't in the Text's allowed list perhaps fix the color or add this on to list?

Reply:
we'd have to talk to @cecileboucheron

Reply:
This whole screen needs to be revamped with a white background.